.env.example

```diff
@@ -12,3 +12,6 @@ LINEAR_OAUTH_SECRET="abc123"
 GITHUB_OAUTH_SECRET="abc123"
 NEXT_PUBLIC_GITHUB_OAUTH_ID=""
 NEXT_PUBLIC_LINEAR_OAUTH_ID=""
+## Created from the Linear Application Settings page, under "Admin Actions: Create developer token"
+## Used to post anonymous comments to Linear
+LINEAR_APPLICATION_ADMIN_KEY=""
```

Comment:
Can this be done in another way? Using some existing functionality?

Reply:
This makes sense for self-hosting teams! As for platform users, I wonder how we could fetch this in a clean UX?

Another option would be to comment as the first-authed (admin) member of a team, like this:

![image](https://user-images.githubusercontent.com/36117635/207083204-d2cbc399-0118-42a3-9687-b68c4cf535d9.png)